Libraries/Text/TextInput/Multiline/RCTUITextView.h

```diff
@@ -26,6 +26,8 @@ NS_ASSUME_NONNULL_BEGIN
 @property (nonatomic, assign, readonly) BOOL textWasPasted;
 @property (nonatomic, copy, nullable) NSString *placeholder;
 @property (nonatomic, strong, nullable) UIColor *placeholderColor;
+@property (nonatomic, readwrite, nullable) NSString *currentAccessibilityError;
```

Comment:
@fabriziobertoglio1987 I'm getting a build error because this differs from Libraries/Text/TextInput/Singleline/RCTUITextField.h

Re
```
stderr: In file included from xplat/js/react-native-github/React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm:15:
xplat/js/react-native-github/Libraries/Text/TextInput/Singleline/RCTUITextField.h:32:54: error: 'retain (or strong)' attribute on property 'accessibilityErrorMessage' does not match the property inherited from 'RCTBackedTextInputViewProtocol' [-Werror,-Wproperty-attribute-mismatch]
[@property](https://www.internalfb.com/intern/profile/property) (nonatomic, readwrite, nullable) NSString *accessibilityErrorMessage;
```

Which property declaration should we use? cc @blavalla if you have thoughts

Reply:
@lunaleaps The RCTTextInputComponentView.mm is the iOS Fabric Component for TextInput. Creates an instance _backedTextInput of type [BackedTextInputViewProtocol](https://github.com/facebook/react-native/blob/f0d0186ebad7312cdcd54b74d8d47e47bd4d741b/Libraries/Text/TextInput/RCTBackedTextInputViewProtocol.h#L20) using RCTUITextView or RCTUITextField constructors.
https://github.com/facebook/react-native/blob/f0d0186ebad7312cdcd54b74d8d47e47bd4d741b/React/Fabric/Mounting/ComponentViews/TextInput/RCTTextInputComponentView.mm#L78

BackedTextInputViewProtocol, RCTUITextView and RCTUITextField headers should match (currently they don't, accessibilityErrorMessage does not exist in RCTBackedTextInputViewProtocol).
Sorry for the mistake. I'm rebasing, building, testing and pushing the commit soon. Thanks

<!--https://stackoverflow.com/questions/2255861/property-retain-assign-copy-nonatomic-in-objective-c-->

Reply:
https://github.com/facebook/react-native/pull/33468/commits/9d1adecb7a58852f50ba43cc758f5e0d19b81e2d

https://user-images.githubusercontent.com/24992535/205132148-e7024469-1f9c-4c57-bfcf-476d21bbf5db.mp4